docs/manual/common.py

```diff
@@ -1,9 +1,16 @@
+import inspect
```

Comment:
Can we organise this folder a bit better by putting all the Python files into their own folder?

Reply:
if I do that then we need a bunch of path hacks to make python find this module. I already had to do it once in the api-reference subdir, since it's not in the same dir. I would like to avoid that hack in the other qmd files as well. What do you think?

Reply:
*api-usage, sorry

Reply:
fixed